### ISSUE TITLE
Make it possible to switch between analysis and coding

### ISSUE THREAD

**Issue by jankapunkt** [openqda/openqda#32]
Requirements:

During analysis as a coder I need to be able to review a coded section "in context" of the rest of the text.
This might involve multiple "jumping" between coding and analysis.
This process is very iterative in the beginning.

Need to discuss this with @LeilaMatayeva

**Comment by jankapunkt** [comment]
We need to preserve the state of the Analysis page and Coding page in order to efficiently switch between them.
This might bring a high level of complexity and we have basically two approaches:

- save the state in the DB on the server
- save the state in localStorage on the client
- save the state in sessionStorage on the client

Needs investigation about the pros/cons of each

**Comment by kadewe** [comment]
We need to discuss this in more detail. Maybe people don't want to switch all the time and it could be beneficially, if one could see in CODING view all other coded text snippets of a code, or in ANALYSIS more text context for a coded text such as 2 lines above and 2 lines below.

**Comment by jdittrich** [comment]
I agree very much with the usecase described by @jankapunkt – when I analyze data, what is in openqda "coding" and "analysis" is one process, in which just I use different features (i.e. assigning codes/retrieving codes) of the software.
